VERSION

```diff
@@ -1 +1 @@
-1.230.0
\ No newline at end of file
+1.230.0//x-release-please-version
```

Comment:
Since release-please is now keeping the package.json file version synced, is there a reason to keep this file around any more? Couldn't we just update all the scripts to use package.json for the version number and update the npm release process to temporarily reset the package.json version number for just the npm publish?

Reply:
I didn't know the full rationale for using a VERSION file. If the only purpose was to allow divergence at build time for NPM, I agree we don't need it. Right now the change to pre-release format is being handled by version-sync directly in the package.json file. We would probably need to move that sync logic into each of the build processes that rely on it (npm publish and npm test-builds). Let me know what you'd like to do.

Reply:
@metal-messiah do you know if any downstream team rely on the VERSION file from our repo? The sync logic is only needed in the npm publish workflows.

Reply:
AFAIK the repo's that I know about that linked to that file have been updated.. There could be others that I don't know about but without our realm of understanding it should be ok.